web/templates/recipient_row.hbs

```diff
@@ -76,7 +76,12 @@
             {{/if}}
 
             {{! Topic menu }}
-            <i class="zulip-icon zulip-icon-more-vertical recipient-row-topic-menu recipient-bar-control recipient_bar_icon" tabindex="0" aria-label="{{t 'Topic actions menu' }}" aria-haspopup="true" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" data-topic-url="{{topic_url}}"></i>
+            {{!-- We define the recipient-row-topic-menu class in a wrapper span
+            since the icon button component already has a tippy tooltip attached
+            to it and attaching the topic actions menu popover to it causes buggy behavior. --}}
```

Comment:
"to it and Tippy does not support multiple tooltips on a single element." would be more precise.

Reply:
Updated, as per the suggestion.